src/components/calculator.js

```diff
@@ -1,34 +1,57 @@
     return (
       <div>
-        <div className="screen">0</div>
+        <div className="screen">
+          {' '}
+          {resultSreen || 0}
+          {' '}
+        </div>
         <div className="buttons">
-          <button type="button">AC</button>
-          <button type="button">+/-</button>
-          <button type="button">%</button>
-          <button type="button" className="orange-color">÷</button>
-          <button type="button">7</button>
-          <button type="button">8</button>
-          <button type="button">9</button>
-          <button type="button" className="orange-color">x</button>
-          <button type="button">4</button>
-          <button type="button">5</button>
-          <button type="button">6</button>
-          <button type="button" className="orange-color">-</button>
-          <button type="button">1</button>
-          <button type="button">2</button>
-          <button type="button">3</button>
-          <button type="button" className="orange-color">+</button>
-          <button type="button" className="wide">0</button>
-          <button type="button">.</button>
-          <button type="button" className="orange-color">=</button>
+          <button type="button" onClick={(e) => { this.handleClick(e); }}>AC</button>
+          <button type="button" onClick={(e) => { this.handleClick(e); }}>+/-</button>
+          <button type="button" onClick={(e) => { this.handleClick(e); }}>%</button>
+          <button type="button" className="orange-color" onClick={(e) => { this.handleClick(e); }}>÷</button>
+          <button type="button" onClick={(e) => { this.handleClick(e); }}>7</button>
+          <button type="button" onClick={(e) => { this.handleClick(e); }}>8</button>
+          <button type="button" onClick={(e) => { this.handleClick(e); }}>9</button>
+          <button type="button" className="orange-color" onClick={(e) => { this.handleClick(e); }}>x</button>
+          <button type="button" onClick={(e) => { this.handleClick(e); }}>4</button>
+          <button type="button" onClick={(e) => { this.handleClick(e); }}>5</button>
+          <button type="button" onClick={(e) => { this.handleClick(e); }}>6</button>
+          <button type="button" className="orange-color" onClick={(e) => { this.handleClick(e); }}>-</button>
+          <button type="button" onClick={(e) => { this.handleClick(e); }}>1</button>
+          <button type="button" onClick={(e) => { this.handleClick(e); }}>2</button>
+          <button type="button" onClick={(e) => { this.handleClick(e); }}>3</button>
+          <button type="button" className="orange-color" onClick={(e) => { this.handleClick(e); }}>+</button>
+          <button type="button" className="wide" onClick={(e) => { this.handleClick(e); }}>0</button>
+          <button type="button" onClick={(e) => { this.handleClick(e); }}>.</button>
+          <button type="button" className="orange-color" onClick={(e) => { this.handleClick(e); }}>=</button>
         </div>
```

Comment:
- [ ] Great job so far, its better to create a button component and map the buttons we want so that we don't repeat the same code and also it will help us practice passing props 👍.